test/system/components/editor_test.rb

```diff
@@ -120,13 +135,62 @@ class EditorTest < ApplicationSystemTestCase
         sign_in!(user)
         visit edit_track_exercise_path(track, exercise)
 
+        # Make sure component is mounted
+        sleep 0.5
         click_on "Feedback"
 
-        assert_text "request mentoring"
+        # in JSX &nbsp; is used and this is thats equivalent here
+        assert_text "Please \nrequest mentoring\n to get feedback."
+      end
+    end
+
+    test "feedback shows automated feedback when only representer" do
+      user = create :user
+      mentor = create :user
+      track = create :track
+      exercise = create :concept_exercise, track: track
+      create :user_track, track: track, user: user
+      solution = create :concept_solution, user: user, exercise: exercise
+      submission = create :submission, solution: solution,
+        tests_status: :passed,
+        representation_status: :generated,
+        analysis_status: :completed
+      create :submission_test_run,
+        submission: submission,
+        ops_status: 200,
+        raw_results: {
+          version: 2,
+          status: "pass",
+          tests: [{ name: :test_a_name_given, status: :pass, output: "Hello" }]
+        }
+      create :iteration, solution: solution, submission: submission, idx: 1
+      create :submission_file, submission: submission
+
+      create :exercise_representation,
+        exercise: exercise,
+        source_submission: submission,
+        feedback_author: mentor,
+        feedback_markdown: "Good job",
+        feedback_type: :essential,
+        ast_digest: "AST"
+      create :submission_representation, submission: submission, ast_digest: "AST"
+
+      use_capybara_host do
+        sign_in!(user)
+        visit edit_track_exercise_path(track, exercise)
+
+        sleep 0.5
+        click_on "Feedback"
+        refute_text "Mentoring Discussion"
+        assert_text "Automated Feedback"
+        # click_on can only click on links or buttons
+        find("details", text: "Automated Feedback").click
+        refute_text "Our Ruby Analyzer has some comments"
+        assert_text "Good job"
```

Comment:
Maybe do something similar like the analyzer comment:

```suggestion
        assert_text "Some representer feedback"
```